cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -17,9 +17,16 @@ using namespace IceInternal;
 
 namespace
 {
+    enum OperationMode
```

Comment:
The name is confusing since we already use OperationMode for something else in Ice. We should also use `enum class` in new C++ code.

What about:
```cpp
enum class MethodKind { SyncInvocation, AsyncInvocation, Dispatch };
```

?